test/infrastructure/container/interface.go

```diff
@@ -97,8 +99,8 @@ type RunContainerInput struct {
 	// IPFamily is the IP version to use.
 	IPFamily clusterv1.ClusterIPFamily
 	// RestartPolicy to use for the container.
-	// If not set, defaults to "unless-stopped".
-	RestartPolicy string
+	// If not set, defaults to RestartPolicyUnlessStopped.
+	RestartPolicy dockercontainer.RestartPolicyMode
```

Comment:
I could not find any usage for `RestartPolicy` in **public** repositories

Reply:
It's fine, see: https://github.com/kubernetes-sigs/cluster-api/blob/f13793dcc1349b5e79f3f7be0737031763d91003/test/README.md#L5